File: sdk/js/PicarioXPO.RenderAPI.V2/UrlGenerators/XpoUrlColor.ts
class XpoUrlColor {
    private color: string;

    constructor(color: string) {
        this.color = color;
    }

    /*
     * Represents a color from a Picario scene for the XPO URL generator
     */
    getColor(): string { return this.color; }
    setColor(val: string) { this.color = val; }
}

File: sdk/c#/PicarioXPO.RenderAPI/XpoUrlColor.cs
namespace PicarioXPO.RenderAPI
{
    /// <summary>
    /// Represents a color from a Picario scene for the XPO URL generator
    /// </summary>
    public sealed class XpoUrlColor
    {
        /// <summary>
        /// Gets or sets the color for this object
        /// </summary>
        public string Color { get; set; }

        /// <summary>
        /// Gets or sets the gloss of this object
        /// </summary>
        public double Gloss { get; set; }
    }
}
